doc/source/serve/deployment-graph.ipynb

```diff
@@ -0,0 +1,508 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "32ef8771",
+   "metadata": {},
+   "source": [
+    "\n",
+    "(serve-deployment-graph)=\n",
+    "\n",
+    "# Serve Deployment Graph\n",
```

Comment:
this should be called `Deployment Graphs` not `Serve Deployment Graph`, otherwise it's redundant in the sidebar